config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,37 @@
+storage-bucket-iam
+==================================================
+
+# NAME
+
+  storage-bucket-iam
+
+# SYNOPSIS
+
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account with the following roles:
+
```

Comment:
Buckets need to be set to be explicitly managed with IAM for this solution to work... The README should include more detail about this

Reply:
@BrianPi , can you please add more details on what needs to be added to instructions. @trodge can you please make sure that the change is OK's by @BrianPi before resolving.

Reply:
This looks fine to me now.